Content.Server/GameObjects/Components/Interactable/ToolComponent.cs

```diff
@@ -15,8 +15,14 @@
 
 namespace Content.Server.GameObjects.Components.Interactable
 {
+    public interface IToolComponent
+    {
+        ToolQuality Qualities { get; set; }
```

Comment:
Setter seems unnecessary and undesirable.

Reply:
The interface was generated from the implementation on line 30. It has a setter, so I added it to the interface. The interface is registered into the component system as an alias, so people can get any IToolComponent instead of WelderComponent. Whoever wrote the original SharedToolComponent wanted the property to be publicly settable, so I was just going along with that.

Reply:
The interface was generated from the implementation on line 30. It has a setter, so I added it to the interface. The interface is registered into the component system as an alias, so people can get any IToolComponent instead of WelderComponent. Whoever wrote the original SharedToolComponent wanted the property to be publicly settable, so I was just going along with that.

Reply:
It *probably* shouldn't be public set but paging @Zumorica.

Reply:
The only reason it needed to be a public setter was for multi tools such as the power drill and the jaws of life.